### ISSUE TITLE
SyntaxError: Unexpected token U in JSON at position 0 

### ISSUE THREAD

**Issue by alexouzounis** [harbur/kubernetic-issues#24]
My kube config is failing to load and in the console i see:

```
reloading
main.controller.js:1 my-context
kubectl.service.js:1 
Requestauth: Objectdomain: "https://localhost:4443/api/v1/"strictSSL: true__proto__: ObjectcallbackFunction: (primise, callback)constructor: (conf)delete: (url, json, done)get: (url, done)getRequestOptions: (path, opts)log: (url, done)patch: (url, body, done)post: (url, body, done)put: (url, body, done)watch: (url, message, exit, timeout)__proto__: Object
```
the next part repeated many times
```
SyntaxError: Unexpected token U in JSON at position 0
    at JSON.parse (<anonymous>)
    at /Applications/Kubernetic.app/Contents/Resources/app.asar/node_modules/k8s/index.js:214:48
    at Array.forEach (native)
    at Request.<anonymous> (/Applications/Kubernetic.app/Contents/Resources/app.asar/node_modules/k8s/index.js:211:46)
    at emitOne (events.js:101:20)
    at Request.emit (events.js:188:7)
    at IncomingMessage.<anonymous> (/Applications/Kubernetic.app/Contents/Resources/app.asar/node_modules/k8s/node_modules/request/request.js:1043:12)
    at emitOne (events.js:96:13)
    at IncomingMessage.emit (events.js:188:7)
    at IncomingMessage.Readable.read (_stream_readable.js:381:10) true
```
then i see:
```
Uncaught (in promise) Unauthorized
_tickCallback @ internal/process/next_tick.js:103
```
and repeat.

My kube config:
```
apiVersion: v1
clusters:
  - cluster:
      certificate-authority-data: >-
        <REDACTED>
      server: 'https://localhost:4443'
    name: kubernetes
contexts:
  - context:
      cluster: kubernetes
      user: admin
    name: my-context
current-context: my-context
kind: Config
preferences: {}
users:
  - name: admin
    user:
      token: <REDACTED>
```

- OSX 10.12.3
- kubectl works perfectly
- version `0.7.1`

**Comment by bviolier** [comment]
I got the same issue, but posted it on: https://github.com/harbur/kubernetic-issues/issues/7

**Comment by spiddy** [comment]
Token-based authentication is now implemented on v0.8.0

https://github.com/harbur/kubernetic-issues/blob/master/CHANGELOG.md#v080.

Parsing of `kube.cfg` for authentication is open-sourced to https://github.com/Goyoo/node-k8s-client/ and will be integrated in next release (v0.9.0)
